doc/classes/AudioStream.xml

```diff
@@ -34,6 +34,11 @@
 				Return the controllable parameters of this stream. This array contains dictionaries with a property info description format (see [method Object.get_property_list]). Additionally, the default value for this parameter must be added tho each dictionary in "default_value" field.
 			</description>
 		</method>
+		<method name="_get_sampling_rate" qualifiers="virtual const">
+			<return type="int" />
+			<description>
+			</description>
+		</method>
```

Comment:
```suggestion
		<method name="_get_sampling_rate" qualifiers="virtual const">
			<return type="int" />
			<description>
				Return the sampling rate of the audio stream in samples per second.
			</description>
		</method>
```

Reply:
Although https://github.com/godotengine/godot/pull/86958 has yet to be merged, if you want to be consistent:
```suggestion
		<method name="_get_sampling_rate" qualifiers="virtual const">
			<return type="int" />
			<description>
				Overridable method. Should return the sampling rate of this audio stream, in samples per second.
			</description>
		</method>
```